### ISSUE TITLE
Installation fail with symfony 4 and flex

### ISSUE THREAD

**Issue by jorge07** [prooph/service-bus-symfony-bundle#55]
Composer require logs: 
```
Problem 1
    - Installation request for prooph/service-bus-symfony-bundle ^0.5.1 -> satisfiable by prooph/service-bus-symfony-bundle[v0.5.1].
    - Conclusion: remove symfony/config v4.0.2
    - Conclusion: don't install symfony/config v4.0.2
    - prooph/service-bus-symfony-bundle v0.5.1 requires symfony/config ~3.3 -> satisfiable by symfony/config[v3.3.0, v3.3.1, v3.3.10, v3.3.11, v3.3.12, v3.3.13, v3.3.14, v3.3.2, v3.3.3, v3.3.4, v3.3.5, v3.3.6, v3.3.7, v3.3.8, v3.3.9, v3.4.0, v3.4.1, v3.4.2].

```

### composer.json
```
"require": {
        "php": "^7.1.3",
        "ext-iconv": "*",
        "api-platform/api-pack": "^1.0",
        "beberlei/assert": "^2.8",
        "ramsey/uuid": "^3.7",
        "symfony/console": "^4.0",
        "symfony/flex": "^1.0",
        "symfony/framework-bundle": "^4.0",
        "symfony/lts": "^4@dev",
        "symfony/yaml": "^4.0"
```

**Comment by gubler** [comment]
The pull request that adds Symfony 4 support (#52) was merged into HEAD and is just waiting to be added to a new version.

**Comment by jorge07** [comment]
Great, thanks!

**Comment by UFOMelkor** [comment]
I have released v0.6.0, this bundle should now support Symfony 4.
